engine/src/main/java/org/terasology/rendering/dag/nodes/OpaqueBlocksNode.java

```diff
@@ -199,4 +215,35 @@ private void renderChunkBoundingBox(RenderableChunk chunk, Vector3f chunkPositio
 
         GL11.glPopMatrix(); // Resets the matrix stack after the rendering of a chunk.
     }
+
+    @Override
+    public void propertyChange(PropertyChangeEvent event) {
```

Comment:
This block of code seems to be absolutely identical to one in a previous class in this commit. Can they inherit it from a common ancestor?